### PR TITLE
arp bug fix

### PR DESCRIPTION
**Pull request by chrhobbs**
more commits coming

### CODE REVIEW

**Comment by chrhobbs** [comment]
Completed all bug fix updates.

File: nxos/arp.py [hunk 1]
```diff
@@ -26,15 +26,29 @@ def get_iparp(message):
...
     try:
         arplist = response['result']['body']['TABLE_vrf']['ROW_vrf']['TABLE_adj']['ROW_adj']
 
     except KeyError:
         arplist = []
 
     if len(arplist) == 0:
-        response2spark = "sorry...no arp entries exist!"
+        response2spark = "# sorry...no arp entries exist!"
+    elif isinstance(response['result']['body']['TABLE_vrf']['ROW_vrf']['TABLE_adj']['ROW_adj'],dict):
```
Comment (kecorbin) [inline nxos/arp.py:41]: don't you already have this stored as arplist?

File: nxos/arp.py [hunk 2]
```diff
@@ -26,15 +26,29 @@ def get_iparp(message):
...
+        # check for dictionary, handle single ARP entry
+        response2spark = "# here is your arp table...\n"
+        arp = response['result']['body']['TABLE_vrf']['ROW_vrf']['TABLE_adj']['ROW_adj']
```
Comment (kecorbin) [inline nxos/arp.py:44]: seems a bit redundant here

**Review by kecorbin — COMMENTED** [review]
overall looks pretty good.  maybe some cosmetic cleanup

File: nxos/arp.py (hunk 2 above)
Comment (chrhobbs) [reply nxos/arp.py:44]: agree. I've cleaned it up.

**Pull request merged**